akka-docs/rst/java/http/routing-dsl/directives/alphabetically.rst

```diff
@@ -119,7 +119,7 @@ Directive                                   Description
                                             ``immutable.Seq[HttpHeader] ⇒ immutable.Seq[HttpHeader]`` function
 :ref:`-mapRouteResult-java-`                     Transforms the ``RouteResult`` with a ``RouteResult ⇒ RouteResult``
                                             function
-:ref:`-mapRouteResultStage-java-`               Transforms the ``RouteResult`` future with a
+:ref:`-mapRouteResultFuture-java-`               Transforms the ``RouteResult`` future with a
```

Comment:
Didn't I just dot that? Maybe I forgot to include it in the PR (multi-line-editing 👍 btw)